authentication/identity_providers/configuring-htpasswd-identity-provider.adoc

```diff
@@ -4,13 +4,34 @@ include::modules/common-attributes.adoc[]
 :context: configuring-htpasswd-identity-provider
 toc::[]
 
-Configure the `htpasswd` identity provider to validate user names and passwords
-against a flat file generated using
-link:http://httpd.apache.org/docs/2.4/programs/htpasswd.html[`htpasswd`].
+[id="identity-provider-overview-{context}"]
+= About identity providers in {product-title}
```

Comment:
I think this is doing something funky. When I build I get this error: 

asciidoctor: ERROR: <stdin>: line 8: level 0 sections can only be used when doctype is book

I *think* you just need to bump it down to level 2.

Reply:
You were correct; this has been corrected.